README.md

```diff
@@ -189,7 +189,6 @@ To use the Google Chromecast support, use the `updateCastContext` method of the
         ...
     }
 ```
-
 # Bitmovin analytics
```

Comment:
I would suggest to put double hashtag `##` so the index will be created correctly